pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +296,85 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+// TestApplyTPR uses apply commad with ThirdPartyResources.
+func TestApplyTPR(t *testing.T) {
```

Comment:
I understand. The unit test implementations for applying the entires for the ThirdPartyResources are on their way. 
We have seen some flaky behavior with respect to the Kind: ThirdPartyResource objects too. 
There were unexpected behavior if there are multiple Kind: ThirdPartyResource objects. Deletion and recreation of them caused unexpected behavior. Those were in v1.3.9.
To reproduce more use cases of ThirdPartyResources, I hope you can see that this unit test still adds value.